src/nvim/eval.c

```diff
@@ -10785,14 +10790,8 @@ static void f_jobstart(typval_T *argvars, typval_T *rettv)
     }
   }
 
-  job_start(opts, &rettv->vval.v_number);
-
-  if (rettv->vval.v_number <= 0) {
-    if (rettv->vval.v_number == 0) {
-      EMSG(_(e_jobtblfull));
-    } else {
-      EMSG(_(e_jobexe));
-    }
+  if (!common_job_start(opts, rettv)) {
```

Comment:
Could be just `common_job_start(opts, rettv);` or was this done because of the return value?